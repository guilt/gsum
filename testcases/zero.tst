Name: zero
Description: Test 1M zero-filled file with default and sha1
File: test.txt 1M zero
AlgoMatrix: default, sha1
