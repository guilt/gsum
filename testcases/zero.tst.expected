Hash-default: 30e14955ebf1352266dc2ff8067e68104607e750abb9d3b36582b8af909fcb58
Hash-sha1: 3b71f43ff30f4b15b5cd85dd9e95ebc7e84eb5a3
